### ISSUE TITLE
The provision button seems to not be changing color when provision is needed

### ISSUE THREAD

**Issue by Orrison** [Orrison/Larval#10]
The provision button should change background color when a provision is needed. Seems to not be doing it anymore.

**Comment by mcastre** [comment]
I can take a look at this for you!

**Comment by Orrison** [comment]
Please do!

**Comment by mcastre** [comment]
Hey @Orrison, would you mind helping me track down this issue? I got the app installed and set up Homestead. I see the Provision button (misspelled by the way haha). What color should the button be when provision is needed? Is it line 42 in Vagrant.js where you are expecting the .is-primary class to be applied? Also, how can i run the app locally and see changes?

Thanks for the help!

**Comment by Orrison** [comment]
Hey @mcastre yeah so line 42 of Vagrant.js would be a start. The class is set dynamically based off that prop so it seems that for some reason it is not getting set properly when a change is made to a box.

It most likely has something to do with the `submitCreateNew` method. On line 385 of App.js you can see code that seems to be setting the state of shouldProvision again when it probably shouldn't be.

It should be setting the should_provision setting like what is happening in the `submitHomesteadSettings` method. Around line 463 of App.js

To get it running in a test environment first run
```
npm install
```
From then on you can load up a local version with
```
npm start
```

**Comment by mcastre** [comment]
@Orrison im unable to run it locally. I keep getting this random error on npm start:
```
/Users/martincastre/.nvm/versions/node/v10.16.0/lib/node_modules/nf/nf.js:48
  if (!/^-/.test(a)) throw new Error("invalid argument " + a);
                     ^

Error: invalid argument start
    at /Users/martincastre/.nvm/versions/node/v10.16.0/lib/node_modules/nf/nf.js:48:28
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (/Users/martincastre/.nvm/versions/node/v10.16.0/lib/node_modules/nf/nf.js:47:14)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)
    at startup (internal/bootstrap/node.js:283:19)
```

Have you encountered that before?

**Comment by Orrison** [comment]
@mcastre hmm, I have not seen that issue before.

What OS are you using? Do you have node.js installed on your system?

**Comment by Orrison** [comment]
Closed by #12
